docs/experiments-uniir-mbeir.md

```diff
@@ -7,10 +7,22 @@ This guide contains instructions for running baselines on the CIRR dataset (one
 ## Data Prep
  
 First, download the CIRR dataset from [here](https://huggingface.co/datasets/TIGER-Lab/M-BEIR/blob/main/cand_pool/local/mbeir_cirr_task7_cand_pool.jsonl) to the `collections/m-beir/CIRR` folder inside pyserini.
+```bash
```

Comment:
nit - add extra newline before triple backtick and remove empty lines... (here and below)